### ISSUE TITLE
加载checkpoint失败

### ISSUE THREAD

**Issue by axuan731** [wenet-e2e/wespeaker#209]
您好：
感谢您的贡献，想请教一些关于训练的问题。
我在训练到46轮的时候停止了，这个时候准确率是18%
![1](https://github.com/wenet-e2e/wespeaker/assets/89143813/98c4576c-0dba-4e96-b8bc-f086e2c2c590)
然后我尝试从第47轮开始训练，这个时候准确率下降非常多(包括之前加载准确率60%的模型，却变成了40%)
![2](https://github.com/wenet-e2e/wespeaker/assets/89143813/19856f7b-eac0-44f7-bea9-92d7a60297db)
并且这个模型不能更新到下一轮，会因为超时报错
![3](https://github.com/wenet-e2e/wespeaker/assets/89143813/4f34649c-d7dc-4dd1-9afb-bfdcd5c06db6)
我是在stage3的时候加入命令行 --checkpoint /.../model_46.pt
请问这个问题出在哪里了？

此外，在LMF的阶段，也会出现90%准确率变成70%的情况，但是依然可以更新到下一轮，EER也会有所下降。
十分感谢您的帮助~

**Comment by JiJiJiang** [comment]
Acc 18% at epoch 46 is too low. What is the Acc after epoch 20?

**Comment by axuan731** [comment]
> Acc 18% at epoch 46 is too low. What is the Acc after epoch 20?

您好，epoch 20开始的准确率是62，epoch 20结束的准确率是46，epoch 21开始的准确率是40，epoch28的准确率最低降到了8%

**Comment by JiJiJiang** [comment]
The accuracys are a little bit low. I guess you should have trained the model with a large mount of training data, as your largest batch id is 16036.
Please check your training config first, e.g., batch size, warmup epoch number, etc. The default setup may be not appropriate for your training data.

**Comment by axuan731** [comment]
> The accuracys are a little bit low. I guess you should have trained the model with a large mount of training data, as your largest batch id is 16036. Please check your training config first, e.g., batch size, warmup epoch number, etc. The default setup may be not appropriate for your training data.

感谢回答~

**Comment by YMLLG** [comment]
我也出现了类似的现象，最后您是怎么解决的？
